src/Commands/Package.php

```diff
@@ -181,6 +187,7 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
 		$this->final                = $this->final ?: $input->getOption( 'final' );
 		$this->clear                = $this->clear ?: $input->getOption( 'clear' );
 		$this->default_build_number = $this->default_build_number ?: $input->getOption( 'build_number' );
+		$this->action_before_zip    = $this->action_before_zip ?: $input->getOption( 'before_zip' );
```

Comment:
I am a bit confused here. Since we have `protected $action_before_zip = 'bin/action-before-zip.sh';` already whats the purpose of the `$input->getOption( 'before_zip' );`

Reply:
To allow someone to pass in a param to chance where that file might live in their plugin.

Reply:
Resolved here on a separate direct commit:
https://github.com/the-events-calendar/tut/commit/b9ab5bcfeb61e4a5c29e0ed7a5171559534460bf